docs/user/02-logs.md

```diff
@@ -476,21 +476,29 @@ The maximum amount of LogPipelines is 5.
 
 ## Troubleshooting
 
-### Logs Not Arriving at the Destination
+### No Logs Arrive at the Backend
 
-   Cause: Wrong backend endpoint configuration or authentication credentials are used.
+Cause: Incorrect backend endpoint configuration (e.g., using the wrong authentication credentials) or the backend being unreachable.
 
-   Remedy: Investigate the cause with the following steps:
-   1. Check the `telemetry-fluent-bit` Pods for error logs by calling `kubectl logs -n kyma-system {POD_NAME}`.
+Remedy: 
+- Check the `telemetry-fluent-bit` Pods for error logs by calling `kubectl logs -n kyma-system {POD_NAME}`.
+- Check if the backend is up and reachable.
 
-### Influx Capacity Reaching Its Limit
+### Not All Logs Arrive at the Backend
 
-  Cause: The backend ingestion rate is too small for the data influx.
+Cause: The backend is reachable and the connection is properly configured, but some logs are refused.
 
-  Remedy:
+Remedy:
+- Check the `telemetry-fluent-bit` Pods for error logs by calling `kubectl logs -n kyma-system {POD_NAME}`. If backend is refusing logs limiting the rate, try the options desribed in [Agent Buffer Filling Up](#agent-buffer-filling-up)
```

Comment:
> If backend is refusing logs limiting the rate

Not sure what this means. Should we add "and"?

Reply:
Agree, adding "and" makes it sound better